packages/apps/contracts/EthUnidirectionalPaymentApp.sol

```diff
@@ -2,7 +2,8 @@ pragma solidity 0.5.9;
 pragma experimental "ABIEncoderV2";
 
 import "@counterfactual/contracts/contracts/interfaces/CounterfactualApp.sol";
-import "@counterfactual/contracts/contracts/interfaces/TwoPartyOutcome.sol";
+// solium-disable-next-line
```

Comment:
can this be in a separate PR?

Reply:
Actually most of this PR is not even related to the work needed for the title of the PR, so I can remove the minimal ERC20 stuff from this PR and treat this as a naming update PR.

Reply:
Updated PR to reflect above comment.